src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -77,9 +77,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $filenames = $input->getArgument('filename');
-        $hasStdin = '-' === ($filenames[0] ?? '');
+        $hasStdin = ['-'] === $filenames;
 
-        if ($hasStdin || 0 === \count($filenames)) {
+        if ($hasStdin || !$filenames) {
             if ($hasStdin || 0 === ftell(STDIN)) { // remove 0 === ftell(STDIN) check in 5.0
                 if (!$hasStdin) {
                     @trigger_error('Calling to the "lint:twig" command providing pipe file content to STDIN without passing the dash symbol "-" explicitly is deprecated since Symfony 4.4.', E_USER_DEPRECATED);
```

Comment:
should we trigger a deprecation warning here, or display a warning on STDERR ? I don't think CLI deprecations are visible to users properly.

Reply:
I wondered the same in the original PR and deprecations are now displayed when using `-vv`. We used to call `$io->caution()` explicitly in 2.8, but I didn't find any usage in 3.4 anymore. Thus the current state.